src/patternfly/components/Nav/nav.scss

```diff
@@ -2,183 +2,119 @@
 
 .pf-c-nav {
   --pf-c-nav--Width: #{pf-size-prem(290px)};
+
+  // Set width when nav is child of sidebar to prevent collapsing
+  .pf-l-page__sidebar .pf-c-nav {
```

Comment:
Hmm... I don't think you would want to move this MQ up because it isn't a simple/basic one - it looks like it requires being in the context of `.pf-l-page__sidebar`. We should only move a MQ up if it's simply redefining a var at a breakpoint. We don't want to write CSS selectors in the vars area. Also, this wouldn't work because it's targeting `.pf-c-nav .pf-l-page__sidebar .pf-c-nav`, which won't exist :)

But I'm not even sure this belongs here. Seems like it should be in the page layout instead. And FWIW, I looked at the page layout, and `.pf-l-page__sidebar` controls the width of the nav, so I'm not sure this width is even needed. I removed this width entirely and it seemed to work fine?

Reply:
Oh yes thats my bad, I just realized it shouldn't have been moved up. Also removed the width and works as normal. Makes sense that the parent would control it.